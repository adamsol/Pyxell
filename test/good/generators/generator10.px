
func f(): Float?... def
    yield 0
    yield null
    yield 1/10^20

func g(): Char?...??... def
    yield null

print [...f()]

for x in g() do
    print x is null
