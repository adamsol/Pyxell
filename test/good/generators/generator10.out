[0, null, 1e-20]
true
